src/EnergyPlus/VentilatedSlab.cc

```diff
@@ -975,10 +1002,18 @@ namespace VentilatedSlab {
                 ventSlab.MaxOASchedPtr = ventSlab.MinOASchedPtr;
             }
 
+            //// Add fan to internal component sets array
+            // SetUpCompSets(state,
+            //              CurrentModuleObject,
+            //              ventSlab.Name + "-SYSTEM",
+            //              "UNDEFINED",
+            //              state.dataIPShortCut->cAlphaArgs(25),
+            //              state.dataIPShortCut->cAlphaArgs(23),
+            //              state.dataIPShortCut->cAlphaArgs(24));
```

Comment:
I thought I cleaned these out. I went in circles trying to get all outputs correct and was commenting out things I added trying to get back to a minimal change. Once changes are agreed on these can be removed.